tests/src/onelogin/models/assigned_user_test.py

```diff
@@ -7,10 +7,6 @@
 import datetime
 from dateutil.tz import tzutc
 
-import sys
-if sys.version_info[0] >= 3:
-    unicode = str
```

Comment:
Do we have an understanding of what this was supposed to accomplish?

Reply:
This is there to make the code compatible with both python 2 and 3. I removed it because we're no longer using `unicode` in this module at all. If we wanted to use `unicode` somewhere else we would need this in the module where we use it.